readthedocs/rtd_tests/tests/test_views.py

```diff
@@ -241,3 +241,17 @@ def test_project_admins_can_delete_subprojects_that_they_are_not_admin_of(self):
             '/dashboard/my-mainproject/subprojects/my-subproject/delete/')
         self.assertEqual(response.status_code, 302)
         self.assertTrue(self.subproject not in [r.child for r in self.project.subprojects.all()])
+
+class URLResolution(TestCase):
+    def setUp(self):
+        self.user = new(User, username='foo+bar')
+        self.user.set_password('test')
+        self.user.save()
+
+        self.project = get(Project, slug='my-mainproject')
+        self.client.login(username='foo+bar', password='test')
+
+    def test_project_details_resolve(self):
+        self.project.users.add(self.user)
+        response = self.client.get('/projects/my-mainproject/')
```

Comment:
I think this should test for the user profile `/profiles/{user}/`

Reply:
I have fixed that as well as added some new users for testing!